readme.html

```diff
@@ -1,10 +1,30 @@
 <html>
   <head></head>
   <body>
-    <h1>Code42 Phantom App</h1>
```

Comment:
Removed `h1` because the name of the app gets this tag automatically so it looked like we had a duplicated title in the documentation.